nucliadb_protos/rust/src/nodereader.rs

```diff
@@ -212,7 +212,7 @@ pub struct ParagraphSearchResponse {
     pub fuzzy_distance: i32,
     #[prost(int32, tag="1")]
     pub total: i32,
-    /// 
+    ///
```

Comment:
empty doc string?

Reply:
this is produced by protos, I dont think I can remove it